network-api/networkapi/buyersguide/templates/primary_nav.html

```diff
@@ -0,0 +1,55 @@
+<div id="primary-nav-container">
+  <div class="wrapper-burger">
+    <div class="menu-container">
+      <div class="narrow-screen-menu">
+        <div class="narrow-screen-menu-background">
+          <div class="narrow-screen-menu-container">
+            <div class="social d-flex flex-column">
+              <a class="social-button body-large social-button-fb" href="#TODO">Facebook</a>
+              <a class="social-button body-large social-button-twitter" href="#TODO">Twitter</a>
+              <a class="social-button body-large social-button-email" href="#TODO">Email</a>
+              <a class="social-button body-large social-button-link" href="#TODO">Copy link</a>
+            </div>
+            <div class="donate">
+              <div class="container text-center">
+                <div class="row">
+                  <div class="col">
+                    <h3 class="h3-heading">Help us keep the Internet healthy</h3>
+                    <p>Mozilla is a nonprofit organization fighting for a healthy interent.</p>
+                    <a class="btn btn-donate m-0 w-100" href="https://donate.mozilla.org">Donate</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <div id="primary-nav-links d-flex align-items-center flex-wrap">
+              <div class="wide-screen-menu">
+                <div class="d-flex align-items-center justify-content-between">
+                  <div class="d-flex align-items-center py-3 logo-section">
+                    <a href="/" class="moz-logo mb-0 mr-3"><span class="sr-only">Mozilla</span></a>
+                    <p class="mb-0 h4-heading flex-wrap">*privacy not included</p>
+                  </div>
+                  <div class="social d-flex align-items-center">
```

Comment:
this feels like a duplication of the above code, with minor adjustments. Could we make this fragment its own template with some conditional logic that generates the narrow vs. wide bits?